### PR TITLE
Added Nav Bar and Fragment Pages

### PR DESCRIPTION
**Pull request by ChristopherNavo**
- Navigation Bar Added
- Fragments and corresponding XML files created for Nav Bar to work.
- Color scheme and Strings Res files
NewNav

NewNav2

### CODE REVIEW

File: local.properties [hunk 1]
```diff
@@ -1,10 +1,8 @@
-## This file is automatically generated by Android Studio.
-# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
-#
-# This file should *NOT* be checked into Version Control Systems,
+## This file must *NOT* be checked into Version Control Systems,
```
Comment (Theillusionofthegift) [inline local.properties]: Hey it looks like you commited a local.properties that shouldn't be

**Review by Theillusionofthegift — CHANGES_REQUESTED** [review]
Hey it looks like you commit a local.properties` that shouldn't be

**Review by Theillusionofthegift — APPROVED** [review]
Looks good!

**Review by mfern08 — APPROVED** [review]
I think it looks great! Everything checks out :)

**Review by mfern08 — APPROVED** [review]
I think it looks great! Everything checks out :)

**Pull request merged**